pkg/statistics/handle/types/interfaces.go

```diff
@@ -68,17 +69,23 @@ type StatsUsage interface {
 
 	// Below methods are for index usage.
 
-	// NewSessionIndexUsageCollector creates a new IndexUsageCollector on the list.
+	// NewSessionIndexUsageCollector creates a new Collector on the list.
 	// The returned value's type should be *usage.SessionIndexUsageCollector, use interface{} to avoid cycle import now.
 	// TODO: use *usage.SessionIndexUsageCollector instead of interface{}.
-	NewSessionIndexUsageCollector() interface{}
-
-	// DumpIndexUsageToKV dumps all collected index usage info to storage.
-	DumpIndexUsageToKV() error
+	NewSessionIndexUsageCollector() *indexusage.SessionIndexUsageCollector
```

Comment:
Better to add a new separate interface `IndexUsage` for it, we can do it later. But could you add a comment here, like "TODO: a new separate interface for it".

Reply:
Good! I've moved these functions to an `IndexUsage` interface. Also, I added a TODO for the following `table delta and stats usage`. :beers: